homeassistant/components/reolink/host.py

```diff
@@ -109,7 +111,9 @@ async def async_init(self) -> None:
             )
             enable_rtsp = True
 
-        if not self._api.onvif_enabled and self._onvif_supported:
+        if not self._api.onvif_enabled and (
+            self._onvif_push_supported or self._onvif_long_poll_supported
+        ):
```

Comment:
If you called that local variable I just commented about something like, is_onvif_supported, you can use it here as well :)

Reply:
thanks for the suggestion